defaults/main.yml

```diff
@@ -1,7 +1,10 @@
 couchbase:
-  cluster: localhost:8091
+  adapter: ansible_eth0
```

Comment:
Not sure if it's worth having a default main.yml file as this will be different for most people (specifically the adapter). What are your thoughts @joe-bowman ?

Reply:
I would maybe have a detailed README with a default example.

Reply:
I would suggest `bind_address` rather than `adapter` and have it default to `0.0.0.0` or `localhost`, as `bind_address` seems the more commonly used nomenclature.

Reply:
But definitely the example in README.md :+1:

Reply:
This isn't being used any more so will remove as we are favouring hostnames where possible with an override of user passed IP when needed